pandas/core/indexes/base.py

```diff
@@ -2311,7 +2311,46 @@ def asof_locs(self, where, mask):
 
     def sort_values(self, return_indexer=False, ascending=True):
         """
-        Return sorted copy of Index
+        Return a sorted copy of the index.
+
+        Return a sorted copy of the index, and optionally return the indices
+        that sorted the index itself.
+
+        Parameters
+        ----------
+        return_indexer : bool, default False
+            Should the indices that would sort the index be returned.
+        ascending : bool, default True
+            Should the index values be sorted in an ascending order.
+
+        Returns
+        -------
+        sorted_index : pandas.Index
+            Sorted copy of the index.
+        _as : numpy.ndarray, optional
```

Comment:
I think numpy usually calls this kind of array an `indexer` or `index_array`.

Reply:
Sure, that's better